.github/workflows/test-and-publish-nightly.yml

```diff
@@ -64,11 +64,11 @@ jobs:
           HASH=${GITHUB_SHA:0:7}
           case "${{ matrix.runson.display }}" in
             macos*)
-              brew install gcovr
-              make COMMIT_TAG=$HASH FAT_BINARY=true release coverage -j
+              make COMMIT_TAG=$HASH FAT_BINARY=true release -j
+              make COMMIT_TAG=$HASH FAT_BINARY=false test-cpp -j
```

Comment:
Shouldn't we run `test-cpp` before `release`?

Reply:
I changed the order, although it does not make a big difference: `release` target only prepares a bundle.